docs/community/contributor_ladder.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Contributor Ladder
+sidebar_position: 40
+---
+
+# Contributor Ladder
+
+* [Contributor Ladder](#contributor-ladder)
+    * [Contributor](#contributor)
+    * [Organization Owners](#oras-organization-owners)
+    * [Subproject Owners](#subproject-owners)
+* [Onboarding a New Owner](#onboarding-a-new-owner)
+* [Inactivity](#inactivity)
+* [Involuntary Removal](#involuntary-removal-or-demotion)
+* [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+* [Contact](#contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! 
+This contributor ladder outlines the different contributor roles within the project, 
+along with the responsibilities and privileges that come with them. 
+Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.
+Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into three sections. 
+"Responsibilities" are tasks that a contributor is expected to do. 
+"Requirements" are qualifications a person needs to meet to be in that role, 
+and "Privileges" are rights contributors on that level are entitled to.
+
+### Contributor
+A Contributor directly adds value to the project. 
+Contributions need not be code. 
+Individuals at the Contributor level may be new contributors, 
+or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+    * Follow the project [contributing guide](./contributing_guide.mdx)
+    * Participation in community Slack channel(s)
+* Requirements (one or several of the below):
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Contribute to the documentation
+    * Attend project meetings, take notes
+    * Answer questions from other community members
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
```

Comment:
@deepeshaburse  For the content below, I think we don't need to replicate the content from https://github.com/oras-project/community/blob/main/governance/GOVERNANCE.md but it would be good to add links point to the governance doc. It will reduce duplicated maintenance.